opentech/apply/categories/models.py

```diff
@@ -59,6 +71,15 @@ class MetaCategory(index.Indexed, MP_Node):
     panels = [
         FieldPanel('parent'),
         FieldPanel('name'),
+        FieldPanel('is_archived'),
```

Comment:
Many move this in to the Options section? Or is there reasons to have it separate?

Reply:
Moved under options panel.